docroot/themes/contrib/civictheme/civictheme.provision.inc

```diff
@@ -0,0 +1,758 @@
+<?php
```

Comment:
"tha MEAT"

Reply:
3 parts:
1. Form alter, validation and submit to inject into theme config UI.
2. Core of the provisioner to discover callbacks
3. Actual provisioning callbacks

Reply:
![Uploading CivicTheme_Demo___CivicTheme_Source.png…]()